ncs/sign_script.py

```diff
@@ -22,14 +22,19 @@
 from argparse import ArgumentParser
 from pathlib import Path
 from cryptography.hazmat.primitives import hashes
-from cryptography.hazmat.primitives.serialization import load_pem_private_key
+from cryptography.hazmat.primitives.serialization import load_pem_private_key, load_der_private_key
 from cryptography.hazmat.primitives.asymmetric.utils import decode_dss_signature
 from cryptography.hazmat.primitives.asymmetric import ec
 from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKey
 from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PrivateKey
+from cryptography.hazmat.primitives.asymmetric.ed448 import Ed448PrivateKey
 from collections import defaultdict
 from enum import Enum, unique
 
+#
+# User note:
+#   Rename the file to 'key_private.der' if you are using keys in DER format.
+#
 PRIVATE_KEY = Path(__file__).parent / "key_private.pem"
```

Comment:
any reason this isn't coming from a Kconfig like other key files are?

Reply:
We've discussed this internally. This file is not present in the repo. The signing is not enabled in the default configuration.
This file is currently used by our tests only and will be scraped once nrfutil will add support for manifest signing, thus we do not want to pollute Kconfigs (and provide user APIs) with the temporary solution.
If someone wants to sign manifests in NCS 2.7.0 the advised way is to configure the signing script path to something custom. The custom script may be based on this file to have a working starting point.

Reply:
@tomchy is correct. The Kconfig you proposed was present in my first approach, but after the internal discussion it got scrapped for the above mentioned reasons.